arm-appinsights/2015-05-01/swagger/syntheticMonitor_API.json

```diff
@@ -0,0 +1,402 @@
+            "schema": {
+              "$ref": "#/definitions/SyntheticMonitorListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SyntheticMonitorList": {
+            "$ref": "../examples/SyntheticMonitorList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location"
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "SyntheticMonitorListResult": {
+      "description": "A list of 0 or more Application Insights synthetic monitor definitions.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Set of Application Insights synthetic monitor definitions.",
+          "items": {
+            "$ref": "#/definitions/SyntheticMonitor"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The link to get the next part of the returned list of synthetic monitors, should the return set be too large for a single request. May be null."
+        }
+      }
+    },
+    "SyntheticMonitor": {
+      "description": "An Application Insights synthetic monitor definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "The kind of web test that this synthetic monitor watches. Choices are ping and multistep.",
+          "default": "ping",
+          "enum": [
+            "ping",
+            "multistep"
+          ],
+          "x-ms-enum": {
+            "name": "WebTestKind",
+            "modelAsString": false
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/WebTestProperties"
+        }
+      }
+    },
+    "WebTestProperties": {
+      "description": "Metadata describing a synthetic monitor for an Azure resource.",
+      "required": [
+        "Name",
+        "Kind"
+      ],
+      "properties": {
+        "SyntheticMonitorId": {
```

Comment:
- [x] Done. I took some fiddler traces of how data is sent back from the server and made sure each field is written exactly as it appears on the wire.